metagraph/tests/graph/test_aligner.cpp

```diff
@@ -601,8 +603,13 @@ TYPED_TEST(DBGAlignerTest, align_noise_in_branching_point) {
     auto path = paths[0];
 
     EXPECT_EQ(query.size() - k + 2, path.size());
-    EXPECT_EQ(reference_1 + "T", path.get_sequence());
-    EXPECT_EQ("4=1D7=", path.get_cigar().to_string());
+    if (!path.get_orientation()) {
+        EXPECT_EQ("AAAACTTTTTTT", path.get_sequence());
+        EXPECT_EQ("4=1D7=", path.get_cigar().to_string());
+    } else {
+        EXPECT_EQ("AAAAAAACTTTT", path.get_sequence());
+        EXPECT_EQ("7=1D4=", path.get_cigar().to_string());
```

Comment:
rc of `AAAACTTTTTTT` is `AAAAAAAGTTTT`. Why do you check for `AAAAAAACTTTT` ?

Reply:
Sure, I just noticed this and was curious. Maybe add a comment or something explaining the test?